README.rst

```diff
@@ -18,18 +19,39 @@ Note: ::
     revisions.
 
 
---------------------
-Installation & Usage
---------------------
+*****
+Usage
+*****
+
+Please refer to  `documentation
+<http://aldryn-reversions.readthedocs.org/en/latest/how_to/usage.html>`_. Or
+`Aldryn Reversion documentation index
+<http://aldryn-reversions.readthedocs.org>`_,
+
+
+------------
+Requirements
+------------
+
+* Python 2.6 or 2.7
+* Django 1.6, 1.7 or 1.8
+* django-reversion
+
+**Optional:**
+
+* django CMS 3.0.12 or later
+* django-parler
 
-Aldryn Platform Users
-~~~~~~~~~~~~~~~~~~~~~
 
-TODO
+------------
+Installation
+------------
 
+Most likely you don't need to install this addon by yourself, and it is
+installed as a dependency for other addon.
+But in case if you are a developer or any other reason to install it manually
```

Comment:
How about:
"If you do need to install this addon manually, just follow thses steps:"
?